UI.py

```diff
@@ -237,18 +237,32 @@ def CreateGetLeavesAcrossRangeOptions():
 
     return member_options
 
-def CreateMultipleLeavesEmbed(leaves_group, include_reason):
-    embed = discord.Embed(
-        title = f'Applied Leaves',
-        description = f'<@!{leaves_group[0][0]["member_id"]}> has requested:',
-        colour = 0x4682B4
-    )
-    embed.set_thumbnail(url = os.getenv("Leave_Balance_Link"))
-    embed.add_field(name = '\u200B', value = '\u200B', inline = False)
+def CreateMultipleLeavesEmbed(leaves, include_reason):
```

Comment:
The names `CreateMultipleLeavesEmbed` and `DesignMultipleLeavesEmbed` aren't clear.

`CreateLeavesAcrossRangeEmbed` just reads better, especially since with the nature of embeds and the way we've had this written, you can't expect to reuse this anywhere else. 

Regarding `DesignMultipleLeavesEmbed`, it's not actually "designing" anything, it's simply formatting/adding data into an embed.

Reply:
Done